Frends.ServiceBus.Read/Frends.ServiceBus.Read/Frends.ServiceBus.Read.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	<TargetFrameworks>net6.0</TargetFrameworks>
+	<Version>1.0.0</Version>
+	<Authors>Frends</Authors>
+	<Copyright>Frends</Copyright>
+	<Company>Frends</Company>
+	<Product>Frends</Product>
+	<PackageTags>Frends</PackageTags>
+	<PackageLicenseExpression>MIT</PackageLicenseExpression>
+	<GenerateDocumentationFile>true</GenerateDocumentationFile>
+	<Description>Frends tasks for Service Bus reading.</Description>
+	<PackageProjectUrl>https://frends.com/</PackageProjectUrl>
+	<RepositoryUrl>https://github.com/FrendsPlatform/Frends.ServiceBus</RepositoryUrl>
```

Comment:
Ain't this suppose to be https://github.com/FrendsPlatform/Frends.CrossplatformServiceBus?

Reply:
Forgot to fix this one after getting a new repo.